pkg/scheduler/ugm/manager.go

```diff
@@ -107,9 +107,6 @@ func (m *Manager) IncreaseTrackedResource(queuePath, applicationID string, usage
 		zap.Stringer("resource", usage))
 	groupTracker := m.GetGroupTracker(appGroup)
 	if groupTracker == nil {
-		log.Log(log.SchedUGM).Error("group tracker should be available in groupTrackers map",
```

Comment:
Should we align the behavior of `DecreaseTrackedResource`[0] with this change?

[0] https://github.com/apache/yunikorn-core/blob/97c38a97a55ee9b457c366c11a4ae32348093e2c/pkg/scheduler/ugm/manager.go#L110

Reply:
In that case we're guared by:
```
	if appGroup == common.Empty {
		return
	}
```
In this code path, we already check it inside `ensureGroupTrackerForApp()`.

Reply:
That is interesting. We do the similar check/ensure for `IncreaseTrackedResource`

https://github.com/apache/yunikorn-core/blob/97c38a97a55ee9b457c366c11a4ae32348093e2c/pkg/scheduler/ugm/manager.go#L98

It seems the error message makes sense since `groupTracker` should be existent in the code path?

Reply:
It doesn't have to be there. Before the refactor, we did not have this check because you can call `IncreaseTrackedResource()` on a `nil` group tracker. However I don't like that approach, just want to return on `nil`.

Reply:
Got it. thanks!

Reply:
Sorry for one more question:

> In that case we're guarded by:

why we not adding similar code to `IncreaseTrackedResource` for consistency? It means that is fine to return directly if `appGroup` is empty. Also, we add error log if `appGroup` is NOT empty and `ensureGroupTrackerForApp` is called?

Reply:
Check the latest version.